packages/pnpm-sync-lib/src/interfaces.ts

```diff
@@ -16,6 +16,13 @@ export interface ITargetFolder {
   folderPath: string;
 }
 
+export interface IFileStat {
+  file: string;
```

Comment:
```suggestion
  /**
   * The absolute path to the file.
   */
  file: string;
```

Reply:
Also, since `isFile` might be `false`, perhaps a better name would be `absolutePath` instead of `file`.